runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -4096,16 +4096,15 @@ class MM_CopyForwardSchemeRootClearer : public MM_RootScanner
 				Assert_MM_mustBeClass(_extensions->objectModel.getPreservedClass(&forwardedHeader));
 				env->_copyForwardStats._offHeapRegionsCleared += 1;
 				void *dataAddr = _extensions->indexableObjectModel.getDataAddrForContiguous((J9IndexableObject *)objectPtr);
-				_extensions->largeObjectVirtualMemory->freeSparseRegionAndUnmapFromHeapObject(_env, dataAddr);
+				_extensions->largeObjectVirtualMemory->freeSparseRegionAndUnmapFromHeapObject(_env, dataAddr, objectPtr, _extensions->indexableObjectModel.getDataSizeInBytes((J9IndexableObject *)objectPtr));
 				*sparseHeapAllocation = false;
 			} else {
 				void *dataAddr = _extensions->indexableObjectModel.getDataAddrForContiguous((J9IndexableObject *)fwdOjectPtr);
 				if (NULL != dataAddr) {
 					/* There might be the case that GC finds a floating arraylet, which was a result of an allocation
 					 * failure (reason why this GC cycle is happening).
 					 */
-					_extensions->largeObjectVirtualMemory->updateSparseDataEntryAfterObjectHasMoved(dataAddr, fwdOjectPtr);
-				}
+					_extensions->largeObjectVirtualMemory->updateSparseDataEntryAfterObjectHasMoved(dataAddr, objectPtr,  _extensions->indexableObjectModel.getDataSizeInBytes((J9IndexableObject *)fwdOjectPtr), fwdOjectPtr);				}
```

Comment:
@LinHu2016 Could you create a follow-up change to put the closing brace back on a line by itself, please?

Reply:
Thanks, I have created a PR, which also clean up some others format issues, https://github.com/eclipse-openj9/openj9/pull/21072